ansible/roles/dnsmasq/tasks/main.yml

```diff
@@ -29,8 +29,7 @@
     owner: 'root'
     group: 'root'
     mode: '0755'
-    unsafe_writes: '{{ True if (core__unsafe_writes|d(True if (ansible_local|d() and ansible_local.core|d()
-                       and ansible_local.core.unsafe_writes|d() | bool) else False) | bool) else omit }}'
+    unsafe_writes: '{{ True if (core__unsafe_writes|d(ansible_local.core.unsafe_writes|d()) | bool) else omit }}'
```

Comment:
Well... I'm not sure why I didn't notice this usage before (the default variable is referenced first instead of the fact) but this is essentially redundant.

Actually the use of the `ansible_local.core.*` facts in various roles could probably be replaced to references to variables defined in the inventory directly. The information flow would also be better, instead of getting the value from the remote host via facts, it could be taken from the inventory. The reason I didn't do this is that there wasn't a good variable namespace to reserve for this. But I think that after renaming the `debops` role to `controller`, the `debops__` namespace could be reserved for global variables, like `debops__unsafe_writes` and similar. Then we could drop most of the stuff in the `core` role facts and rely on the users to define the variables in the inventory. Then the above code could be changed to:
```yaml
unsafe_writes: '{{ True if debops__unsafe_writes|bool else omit }}'
```
As for the current cases in the file - I guess you can leave them as they are for now.

Reply:
Sounds good. `debops__` works for me. Then I will leave this as is for now.